java-util/src/main/java/io/druid/java/util/common/io/smoosh/FileSmoosher.java

```diff
@@ -494,6 +498,7 @@ public boolean isOpen()
     public void close() throws IOException
     {
       closer.close();
+      FileSmoosher.LOG.info("Created smoosh file [%s] of size [%s] bytes.", outFile.getAbsolutePath(), outFile.length());
```

Comment:
same reason.... mostly we look at logs afterwards where most jobs would've run without debug level logs.